index.js

```diff
@@ -90,6 +121,14 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+const myArrowFunction = () => {
+    msg2  = "Tomorrow is Wednesday"
```

Comment:
You'l also want to use a keyword here when you instantiate and assign teh variable msg2. (if you don't it default to var which won't break, but isn't recommended). So this could be
`const msg2 = "Tomorrow is Wednesday"`

Since the only thing you're doing with msg2 is returning it, you could also forgo saving the variable to a value if you want, and just return it directly.

```
const myArrowFunction = () => {
  return "Tomorrow is Wednesday"
}
```

Then, if you wanted to make it even more concise, you could write this as
`const myArrowFunction = () => "This is a function expression!";`
^ You're allowed to write it like that when there's nothing else you need to do in the code block beside return. It's sort of a shorthand.